examples/pytorch/text-generation/run_generation.py

```diff
@@ -355,7 +354,9 @@ def main():
     if tokenizer.pad_token is None:
         tokenizer.pad_token = tokenizer.eos_token
     model = model_class.from_pretrained(args.model_name_or_path)
-    model.to(args.device)
+
+    # Prepare model with `accelerator`.
+    accelerator.prepare_model(model)
```

Comment:
```suggestion
    model = accelerator.prepare_model(model)
```

Reply:
Thank you for bringing this to my attention. fixed it.